tools/windows-node-installer/pkg/cloudprovider/factory.go

```diff
@@ -76,6 +76,9 @@ func CloudProviderFactory(kubeconfigPath, credentialPath, credentialAccountID, r
 
 // makeValidAbsPath remakes a path into an absolute path and ensures that it exists.
 func makeValidAbsPath(path string) (string, error) {
+	if path == "" {
```

Comment:
Wouldn't you check the status below? https://github.com/openshift/windows-machine-config-operator/pull/35/files#diff-53cac869da03e2f45bd8b4ebb87ceba1L90 that line should've been first line in this function before you get started with constructing absolute path, you also need to ensure that you're not errorring out just when path doesn't exist.

Reply:
I might need more clarification on that.... if that's ok. Thank you!